CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+project(antares-deps)
+cmake_minimum_required(VERSION 3.0)
+
+set(DEFAULT_BUILD_TYPE Release)
+
+# Set Build Type
+if(NOT CMAKE_BUILD_TYPE)
+    set(CMAKE_BUILD_TYPE ${DEFAULT_BUILD_TYPE})
+endif()
+
+if (DEFINED VCPKG_ROOT)
+    include(${VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake)
+endif()
+
+#CMake function to build dependancy from git
+include("cmake/utils.cmake")
+
+option(BUILD_ALL "Build all dependencies" OFF)
+message(STATUS "Build all dependencies: ${BUILD_ALL}")
+
+option(USE_ORTOOLS_STABLE "Use OR-Tools stable instead of RTE OR-Tools branch with Sirius support" OFF)
+message(STATUS "Use OR-Tools stable instead of RTE OR-Tools branch with Sirius support: ${USE_ORTOOLS_STABLE}")
```

Comment:
No CI with this option but could be used in antares-simulator to use a different OR-Tools branche

Reply:
OK, good for me